src/main/scala/com/ringcentral/cassandra4io/cql/package.scala

```diff
@@ -75,34 +75,36 @@ package object cql {
     def unlogged[F[_]: Functor] = new Batch[F](new BatchStatementBuilder(BatchType.UNLOGGED))
   }
 
+  trait BindableBuilder[P] {
```

Comment:
could it be hidden in `CqlTemplateStringInterpolator` companion object and imported in class explicitly if necessary?

Reply:
It was my first idea and it didn't helped. I guess there is problem with singleton inside class, but haven't found any info related to it and 2.12

Reply:
Sorry, didn't get your question from the first time - thought you are suggesting to import BindableBuilder into CqlTemplate String Interpolation without changing the hierarchy